### DISCUSSION TITLE
1 Network, 2 NAS, 4 PCs, how to manage?

### DISCUSSION THREAD

**Discussion by benkywong** [Chia-Network/chia-blockchain#4881] [Support]
I have a 1000Mbps network(WAN and LAN) and 2NAS (10bays, 8 bays), 4PCs (mostly AMD 1950x with many NVMe SSD plotting)

Which one is the best practice? (I see many "This should be below 5 seconds to minimize risk of losing rewards." in log file)

A. PC-A farm NAS A. PC-B farm NAS B while all PCs plotting
B. PC-A farm NAS A+B while all PCs plotting
C. PC-A farm NAS A w/o plotting. PC-B farm NAS B w/o plotting, let just PC-C and PC-D do plotting

ps. Actually one of the NAS isn't in this network now. It is now a remote NAS. But even the local NAS still pop up "5sec" WARN

**Comment by benkywong** [comment]
Or I should give up all NAS and directly attach all HDD to those PCs? Is that a good idea that all PCs full node farming with their local HDD? What is the benefit of 1full node, other just harvester?

**Comment by benkywong** [comment]
please help

**Comment by cccat6** [comment]
NAS does have the problem of latency. I am also using the net disk, I can say I am lucky to mine 6 XCH but it definitely will be a risk. I have to accept the risk
I usually have full proof time about 27 seconds. I tried a lot to optimize but seems that is the maximum.
Currently, I got 300+TiB of plots and spread them in different stores. I use a network to connect them and merge them as a single virtual drive. It is kind of similar to yours.
So my suggestion is. If you do the testing and the average proof find time is less than 30s. Keep it. If not, I guess you might need to move your drive directly to your harvester PC.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
